content/authors/felix-vaati/index.md

```diff
@@ -2,7 +2,7 @@
 title: Felix Vaati
 type: authors
 interests: PHP & Kotlin
-twitter: https://twitter.com/DyrstiuAppdh
+twitter: https://twitter.com/Dyrstiu
 images:
   - url: /engineering-education/authors/felix-vaati/avatar.png 
```

Comment:
You haven't added your profile image to the `authors` folder.

Reply:
Hi , the image was already there. What i changed was the twitter URL only as you can see from github snippet above